common/country_types/hfe_country_types.txt

```diff
@@ -115,84 +122,117 @@ human_fallen_empire = {
 		}
 	}
 	modules = {
-		all_technology_module = {}
-		# tiered_technology_module = {
-		# 	tier = 5
-		# 	rare = yes
-		# }
+#		all_technology_module = {}
+		tiered_technology_module = {
+			tier = 5
+			rare = yes
+		}
+		basic_technology_module = {}
```

Comment:
Not need this line

Reply:
The Basic Technology Module allows the AI to research in the first place. The Tier Five Technology Module advances them to that level.

Reply:
They don't need to do research before awakening.

Reply:
You can't give it Tier Five technology without that module. If you don't, it'll be like the Vanilla rendition Space Creatures that cannot upgrade.

Reply:
This sound strange, can you show it?

Reply:
I was mistaken. It isn't needed, as the Tiered Technology Module allows for further research.